### PR TITLE
Upgrade devDependencies packages

### PR DESCRIPTION
**Pull request by erdkse**
This PR...

## Changes

-

## Fixes

-

## How to test it

-

## Screenshots

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,72 +22,72 @@
...
+    "@typescript-eslint/parser": "5.30.0",
+    "babel-plugin-styled-components": "2.0.7",
+    "builder-util-runtime": "9.0.2",
+    "concurrently": "7.2.2",
     "craco-alias": "3.0.1",
     "craco-less": "2.0.0",
     "cross-env": "7.0.3",
-    "electron": "19.0.4",
-    "electron-builder": "23.0.4",
+    "electron": "19.0.6",
```
Comment (olensmar) [inline package.json:61]: 19.0.7 was just released - https://github.com/electron/electron/releases/tag/v19.0.7

File: src/components/organisms/ClusterDiffModal/ClusterDiffModal.tsx [hunk 2]
```diff
@@ -357,7 +357,7 @@ function ClusterDiffModal() {
         maxConstraints={[windowSize.width - 64, resizableBoxHeight]}
         axis="x"
         resizeHandles={['w', 'e']}
-        handle={(h: number, ref: LegacyRef<HTMLSpanElement>) => (
+        handle={(h: any, ref: LegacyRef<HTMLSpanElement>) => (
```
Comment (topliceanurazvan) [inline src/components/organisms/ClusterDiffModal/ClusterDiffModal.tsx:360]: Let's use `ResizeHandle` type here

File: src/components/organisms/ClusterResourceDiffModal/ClusterResourceDiffModal.tsx [hunk 3]
```diff
@@ -274,7 +274,7 @@ const ClusterResourceDiffModal = () => {
           maxConstraints={[window.innerWidth - 64, resizableBoxHeight]}
           axis="x"
           resizeHandles={['w', 'e']}
-          handle={(h: number, ref: LegacyRef<HTMLSpanElement>) => (
+          handle={(h: any, ref: LegacyRef<HTMLSpanElement>) => (
```
Comment (topliceanurazvan) [inline src/components/organisms/ClusterResourceDiffModal/ClusterResourceDiffModal.tsx:277]: same as above

File: src/components/organisms/LocalResourceDiffModal/LocalResourceDiffModal.tsx [hunk 4]
```diff
@@ -358,7 +358,7 @@ const DiffModal = () => {
           maxConstraints={[window.innerWidth - 64, resizableBoxHeight]}
           axis="x"
           resizeHandles={['w', 'e']}
-          handle={(h: number, ref: LegacyRef<HTMLSpanElement>) => (
+          handle={(h: any, ref: LegacyRef<HTMLSpanElement>) => (
```
Comment (topliceanurazvan) [inline src/components/organisms/LocalResourceDiffModal/LocalResourceDiffModal.tsx:361]: Same as above

File: src/components/organisms/TemplateModal/TemplateModal.tsx [hunk 5]
```diff
@@ -177,7 +177,7 @@ const TemplateModal: React.FC<TemplateModalProps> = props => {
         maxConstraints={[window.innerWidth - 64, containerHeight]}
         axis="x"
         resizeHandles={['w', 'e']}
-        handle={(h: number, ref: LegacyRef<HTMLSpanElement>) => (
+        handle={(h: any, ref: LegacyRef<HTMLSpanElement>) => (
```
Comment (topliceanurazvan) [inline src/components/organisms/TemplateModal/TemplateModal.tsx:180]: Same as above

**Review by topliceanurazvan — CHANGES_REQUESTED** [review]
Only some types changes, other than that looks good 👍🏻

**Review by topliceanurazvan — APPROVED** [review]

**Pull request merged**